02-peer-protocol.md

```diff
@@ -238,7 +238,12 @@ The receiving node MAY fail the channel if:
   - it considers `max_htlc_value_in_flight_msat` too small.
   - it considers `channel_reserve_satoshis` too large.
   - it considers `max_accepted_htlcs` too small.
-  - it considers `dust_limit_satoshis` too small and plans to rely on the sending node publishing its commitment transaction in the event of a data loss (see [message-retransmission](02-peer-protocol.md#message-retransmission)).
+  - it considers `dust_limit_satoshis` too small and plans to rely on the sending
+    node publishing its commitment transaction in the event of a data loss
+    (see [message-retransmission](02-peer-protocol.md#message-retransmission)).
+    A recommended lower bound is 330 satoshis, which matches the bitcoin network's
```

Comment:
Would be good IMO to add the derivation of this value somewhere as errata. I've seen some discussion lately between developers where both sides start with the same base level knowledge, but end up deriving conflicting values for dust depending on where they apply the witness discount (IIRC).

Reply:
I'll add a section in BOLT 3, I agree that many people will likely find that helpful.

Reply:
I have a pull to btcd to export this info, the body of the pull shows my calculations for various output types: https://github.com/btcsuite/btcd/pull/1742

Reply:
I added details about the calculation in https://github.com/lightningnetwork/lightning-rfc/pull/894/commits/40cfede1b7f325949f8cb565cb724fdd71cf7cfc